src/intl/en/learn-quizzes.json

```diff
@@ -613,5 +613,50 @@
+  "smart-contracts-2-prompt": "What is a metaphor that most closely characterizes the functioning of smart contracts?",
+  "smart-contracts-2-a-label": "A bank",
+  "smart-contracts-2-a-explanation": "Banks require manual execution and are structured as hierarchical entities, whereas smart contracts are executed predictable by computers with unchangeable rules.",
+  "smart-contracts-2-b-label": "A digital vending machine",
+  "smart-contracts-2-b-explanation": "The vending machine will only dispense your desired product after all requirements are met - specific inputs guarantee predetermined outputs. This is similar to the logic of smart contracts.",
+  "smart-contracts-2-c-label": "A calculator",
+  "smart-contracts-2-c-explanation": "smart contract code can be used for calculations, but is not limited to that. Rather, smart contracts are blockchain-based programs that follow an 'if this then that' logic.",
+  "smart-contracts-2-d-label": "A website",
+  "smart-contracts-2-d-explanation": "A website is the frontend which captures user directions. A smart contract is the backend logic where these directions are executed, and the result may be returned.",
+  "smart-contracts-3-prompt": "Which is NOT a main characteristic of smart contracts?",
+  "smart-contracts-3-a-label": "Deterministic execution",
+  "smart-contracts-3-a-explanation": "The main benefit of a smart contract is that it deterministically executes unambiguous code, without human interpretation or bias.",
+  "smart-contracts-3-b-label": "Public record",
+  "smart-contracts-3-b-explanation": "With smart contracts on a public blockchain, anyone can instantly track asset transfers and other related information.",
+  "smart-contracts-3-c-label": "Privacy protection",
+  "smart-contracts-3-c-explanation": "As blockchains are pseudonymous networks, transactions are tied publicly to a unique cryptographic address, not an identity.",
+  "smart-contracts-3-d-label": "Ability to change",
+  "smart-contracts-3-d-explanation": "A smart contract cannot be changed once created—it is guaranteed to execute according to the rules defined by its code.",
+  "smart-contracts-4-prompt": "Which is NOT an application of smart contracts?",
+  "smart-contracts-4-a-label": "Cryptocurrencies",
+  "smart-contracts-4-a-explanation": "Cryptocurrencies are token objects which are defined and tracked using smart contracts.",
+  "smart-contracts-4-b-label": "Protocol changes",
+  "smart-contracts-4-b-explanation": "While protocol changes can sometimes use smart contracts, their creation and definition are proposed through transparent online forums and implemented in client software.",
+  "smart-contracts-4-c-label": "Non-fungible tokens (NFTs)",
+  "smart-contracts-4-c-explanation": "Smart contracts are used to define a wide range of NFTs, ranging from digital art to asset ownership certificates.",
+  "smart-contracts-4-d-label": "Decentralized autonomous organizations (DAOs)",
```

Comment:
Going to patch this as well.. swapping DAO out for "insurance policy" since we don't mention DAOs on the page, but we do mention insurance.